packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,95 @@
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
```

Comment:
Variables we do have. Mixins we do not, so I removed those. https://github.com/cerner/terra-core/pull/280/commits/b8e6bb2d3e15c8d98a5588265375c38b2ffb2a50